airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/DebeziumRecordIterator.java

```diff
@@ -148,7 +148,10 @@ private PgLsn extractLsn(ChangeEvent<String, String> event) {
         .flatMap(source -> Optional.ofNullable(source.get("lsn").asText()))
         .map(Long::parseLong)
         .map(PgLsn::fromLong)
-        .orElseThrow(() -> new IllegalStateException("Could not find LSN"));
+        .orElseThrow(() -> {
+          System.out.println("event = " + event);
```

Comment:
debug statement

Reply:
thanks! still debugging this bit with the delete issue i mentioned.